k8s/openebs-operator.yaml

```diff
@@ -592,7 +615,19 @@ spec:
             - name: OPERATOR_NAME
               value: "node-disk-operator"
             - name: CLEANUP_JOB_IMAGE
-              value: "quay.io/openebs/linux-utils:1.4.0"
+              value: "openebs/linux-utils:3.9"
```

Comment:
This image should be quay.io/openebs/linux-utils:1.9.0 or quay.io/openebs/linux-utils:ci.

Reply:
yes removed the change in earlier commit